config/config.yaml

```diff
@@ -16,6 +15,9 @@ dirs:
 seed: 4242
 
 # Dataset parameters
+characters_to_keep: 'abcdefghijklmnopqrstuvwxyzæøå0123456789éü '
+max_seconds_per_example: 10
+dataloader_num_workers: 8
 dataset_probabilities:  # null = equal probability to every dataset
```

Comment:
Its unclear to me what this comment is trying to tell me :)

Reply:
Just that if you don't specify your dataset probabilities, then you will sample from each dataset with equal probability. E.g., if you have two datasets then it would become [0.5,0.5], and if you have four datasets then it would be [0.25,0.25,0.25,0.25].

Does that make sense?

Reply:
Added comment explaining this now